iroh/src/magicsock/node_map/path_state.rs

```diff
@@ -44,11 +47,15 @@ pub(super) struct PathState {
     /// The time this endpoint was last advertised via a call-me-maybe DISCO message.
     pub(super) call_me_maybe_time: Option<Instant>,
 
-    /// The most recent [`PongReply`].
+    // /// The most recent [`PongReply`].
+    // ///
+    // /// Previous replies are cleared when they are no longer relevant to determine whether
+    // /// this path can still be used to reach the remote node.
+    // pub(super) recent_pong: Option<PongReply>,
```

Comment:
what about this? not needed anymore?

Reply:
Yeah sorry, I'll remove it.

The recent pong is now tracked in `PathValidity`.
`PathValidity` is like a mix between `Option<PongReply>` and what used to be `BestAddr`.